### ISSUE TITLE
Rerun to update configuration fails on "Set heketi-cli command"

### ISSUE THREAD

**Issue by gevorg15** [openshift/openshift-ansible#5232]
#### Description

Building the cluster initially works fine but if you want to make configuration changes and re-run the ansible playbook it fails when setting up heketi for glusters

##### Version
```
* ansible 2.3.2.0

* openshift-ansible-3.6.173.0.17-1-6-g1b39fbb5
```

##### Steps To Reproduce
1. Run playbook with native cluster
2. Run the same playbook and inventory file again

##### Expected Results

```
Expecting the playbook to run successfully over and over again
```

##### Observed Results

```
TASK [openshift_storage_glusterfs : Set heketi-cli command] **************************************************************************************************************************************************************************************************************************************************************************************************************************************
fatal: [node1.domain.tld]: FAILED! => {
    "failed": true
}

MSG:

the field 'args' has an invalid value, which appears to include a variable that is undefined. The error was: 'dict object' has no attribute 'results'

The error appears to have been in '/Users/user/Desktop/openshift/openshift-ansible/roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml': line 192, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Set heketi-cli command
  ^ here


```

**Comment by paralin** [comment]
Broken for me too.

**Comment by paralin** [comment]
To be honest there's not a whole lot that's NOT broken with this setup...

**Comment by dustymabe** [comment]
i'm seeing this as well on fedora 26 atomic host.

**Comment by ttindell2** [comment]
+1. Same issue here. Had to remove GlusterFS completely and rerun to get it to work.

**Comment by jarrpa** [comment]
Best I know, officially, complete idempotence is not guaranteed across all openshift-ansible playbooks. As the GlusterFS-related playbooks are still relatively new, we currently only recommend them for greenfield deployments and not for config updates or brownfield deployments.

That said, https://github.com/openshift/openshift-ansible/pull/5415 should be a step to alleviate this issue. :)

**Comment by jarrpa** [comment]
Fixed by #5415.
